docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -0,0 +1,134 @@
+for instance returns a list of `childItems` that hold for each subcomponent the information that's assumed to be present
+for the start of this very scenario (see Assumptions 1, 2). Thus, BoM and the DT deployment in combination 
+enable recursive browsing of the parts-tree, collecting data along the way.
+
+### 2. Adding a new Twin for a given Asset
+
+| Scenario                                                                                                      | Participants                                                   | Assumptions                             | Links to Use-Cases    |
+|---------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|-----------------------------------------|-----------------------|
+| A party (who is not the supplier of an part) publishes data about a previously uncovered aspect of the asset. | - Data Provider (Supplier) <br/> - Data Provider (Third Party) | 1. Third Party knows an id of the asset | - PURIS (coming soon) |
+
+Let's suppose, a car has reached it end-of-life. The object is sold to a dismantler who disassembles the vehicle and wants
+to publish that exact information. As the Digital Twin is owned by the OEM of the vehicle, it will not be accessible to
+everyone in the network. Thus, the dismantler must publish the data themselves, by deploying:
+
+- Digital Twin Registry (DTR)
+    - registered in a DSP-conformant connector offering the DTR to the Dataspace.
+    - registers Digital Twins (ShellDescriptors in AAS vocabulary) with specificAssetIds including at least one that's
+  equivalent to one attached to the OEM's twin.
+- Submodel (like [_Certificate of Destruction_](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/4889d8482fc6b233eb0f56f3ac94c5ea4004bc05/io.catenax.certificate_of_destruction/1.0.1/gen/CertificateOfDestruction.json))
+    - registered in a DSP-conformant connector to the Dataspace.
+    - registered to the correct ShellDescriptor in the DTR.
+    - Serving arbitrary but well-specified json via `GET` requests from the `/$value` endpoint.
+
+#### 2.1. Discovery by Convention
+
+In certain use-cases, participants can agree on conventions where a specific Submodel may be expected.
+This information can only be provided by the buyer of a certain part who creates a new Twin for the part with identical
+`assetIds`. Such conventions about how to locate certain types of Submodels should be explicitly mentioned in the 
+Standards and Kits for a use-case as the default hypothesis is always that a [Supplier holds relevant data](#1-fetching-a-suppliers-twin).
+
+#### 2.2. Discovery by Registration
+
+Additionally, the dismantler must signal in a [BPN-Discovery-Service](../page_software-operation-view.md) that he indeed has data on said vehicle.
```

Comment:
he or she

Reply:
Via the BPN Discovery Service it is not possible to decide whether the dismantler has information on a specific vehicle. It is only possible to find the BPN of the dismantler so that in a subsequent steps the data consumer can look for twins in the Digital Twin Registry of the Dismantler for this specific vehicle.

Reply:
Right, but the dismantler would not register with a certain assetId if they didn't have data on said asset. For all cars the dismantler does not have data on, they don't register at the BPN Discovery Service.